accounts/admin.py

```diff
@@ -41,11 +41,30 @@ class UserAdmin(admin.ModelAdmin):
     )
 
 
+class MajorAdmin(admin.ModelAdmin):
+    readonly_fields = ("id",)
+    list_filter = ("is_active", "degree_type")
+    list_display = ("name",)
+
+    def name(self, obj):
+        return obj.name
```

Comment:
Ah, if you copied this logic from the StudentAdmin, I don't think we need a custom function for this (and the school one). We had to do that for the Student model in order to get "username" to alias "user.username", but since name is a field on both of these models you probably don't need them here

Reply:
fixed